pycloudlib/ec2/cloud.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, tag, access_key_id=None, secret_access_key=None,
                 'Please configure ec2 credentials in $HOME/.aws/credentials')
 
     def create_vpc(self, name, ipv4_cidr='192.168.1.0/20'):
```

Comment:
(Come back to read this after reading my long comment below.)

It may also make sense for this to move to a `VPC.get_or_create` classmethod, so that all the VPC-related API usage is located in a single class.

Reply:
I might hold off on the get_or_create on VPC class at the moment, I'd like to discuss post-standup tomorrow about incorporating that in the 2nd branch I need to queue up for ec2 support.